Sources/TripKitUI/views/results/TKUITimePickerSheet+Configuration.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  TKUITimePickerSheet+Configuration.swift
+//  TripKitUI-iOS
+//
+//  Created by Brian Huang on 3/11/21.
+//  Copyright © 2021 SkedGo Pty Ltd. All rights reserved.
+//
+
+import Foundation
+
+public extension TKUITimePickerSheet {
+  
+  struct Configuration {
```

Comment:
I'll remove this and instead make this part of the configuration for the `TKUIRoutingResultCard`. The reason for that is that it doesn't make as much sense to apply this to _every_ time picker, regardless of how it's used, and it makes more sense as a configuration of whichever use case this is being used in.